geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeaturePropertyAccessor.scala

```diff
@@ -0,0 +1,33 @@
+/***********************************************************************
+* Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.geotools
+
+import com.typesafe.scalalogging.LazyLogging
+import org.geotools.filter.expression.{PropertyAccessor, PropertyAccessors}
+import org.opengis.feature.simple.SimpleFeature
+
+class SimpleFeaturePropertyAccessor[T] extends LazyLogging {
```

Comment:
Yeah, I converted this to an object and moved the type to the method but had to classTag T since you can't call getClass[T].